static/src/javascripts/projects/journalism/snippet-feedback.js

```diff
@@ -0,0 +1,88 @@
+// @flow
+import fastdom from 'lib/fastdom-promise';
+import mediator from 'lib/mediator';
+import ophan from 'ophan/ng';
+import detect from 'lib/detect';
+
+const SnippetFeedback = (): void => {
+    const snippets = [...document.querySelectorAll('.explainer-snippet--new')];
```

Comment:
Will there ever be more than one snippet? I'm just asking, because scroll listeners are costly.